### DISCUSSION TITLE
Spaceserp showing too many failed keywords

### DISCUSSION THREAD

**Discussion by dangson92** [towfiqi/serpbear#186] [General]
Hi there, anybody using spaceserp that have too many failed keywords after tracking. So how to fast re-tracking the failed keywords or just reduce this failed

**Comment by towfiqi** [comment]
@dangson92 I had high hopes for Spaceserp, but it's very disappointing. Most of the time the requests take so long that it fails.

**Comment by dangson92** [comment]
@towfiqi sure, so many failed keywords. I contact to them to ask about this but no reply so I create this topic to find another way. 

I think we can fix it by add function to quick tracking all failed keywords

**Reply by towfiqi** [reply]
Do you mean force refreshing all the failed keywords? 
Can't you just use the "Refresh All" feature since most of your keyword scrapes failed?

**Reply by dangson92** [reply]
Yes, force refreshing all the failed keywords. Because when I tracking with Spaceserp, My campaign failed ~30-40% so I just want to refreshing the failed keywords. If I using Refresh All, it will refreshing the success keywords

**Reply by towfiqi** [reply]
Did you enable the Auto Rety failed keywords option from the settings panel?

**Reply by dangson92** [reply]
I actived it, but I think this function not working.

**Reply by towfiqi** [reply]
To make sure that the feature is not working, don't manually refresh the failed keywords. SerpBear scrapes the failed keywords every hour, if the scrape fails, you should see a lot of `[ERROR] Updating SERP for Keyword` messages in your docker log.
